aws/data_aws_unity_catalog_assume_role_policy.go

```diff
@@ -0,0 +1,63 @@
+package aws
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+
+	"github.com/databricks/terraform-provider-databricks/common"
+)
+
+func DataAwsUnityCatalogAssumeRolePolicy() common.Resource {
+	type AwsUcAssumeRolePolicy struct {
+		RoleName           string `json:"role_name"`
+		UnityCatalogIamArn string `json:"unity_catalog_iam_arn,omitempty"`
```

Comment:
Should we mark it as `omitempty,computed` to avoid configuration drift when we set it in case of empty one?

Reply:
I set it just in case, tf did not complain when I was testing it manually. data sources never have drifts is my guess